components/tag/index.css

```diff
@@ -12,195 +12,190 @@
  */
 
 .spectrum-Tag {
-	/* TODO - replace placeholder disabled avatar opacity with correct token once avatar is migrated */
-	--spectrum-avatar-opacity-disabled: 0.3;
-
 	--spectrum-tag-animation-duration: var(--spectrum-animation-duration-100);
-	--spectrum-tag-border-width: var(--spectrum-border-width-100);
+
+	/* border color - transparent unless modified or high contrast mode */
+	--spectrum-tag-border-width: var(--spectrum-border-width-100); /* borders remain in WHCM but appear transparent otherwise */
+	--spectrum-tag-border-color: var(--highcontrast-tag-border-color, var(--mod-tag-border-color, transparent));
```

Comment:
Colors have var-stacks at the top level, other properties tend to be numeric or change between sizes, so I left those as-is.